CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
-
 project(rqt_image_view)
+
+add_compile_options(-std=c++11)
```

Comment:
This should make the CI build pass for `xenial`, @dirk-thomas 
I hope this is a valid approach, otherwise I'll have to re-implement things w/o C++11 features.

Reply:
Yes, using C++11 in Kinetic and higher is fine, see [REP 3](http://www.ros.org/reps/rep-0003.html#kinetic-kame-may-2016-may-2021).